### PR TITLE
@arijeetbaruah moved 'Pick Radius' from 'Color' to '3D View' fixes #0004147

### PR DESCRIPTION
**Pull request by arijeetbaruah**
Screencast showing where Pick Radius is located in the m Preferences https://i.imgur.com/N7MFELX.mp4

This is not a 'Color' issue. It's a mouse-related issue.

Case arguing for change:
https://forum.freecadweb.org/viewtopic.php?f=8&t=39650&p=336939#p336932
https://forum.freecadweb.org/viewtopic.php?f=3&t=39161

Forum thread to poll the community for the desired location => https://forum.freecadweb.org/viewtopic.php?f=8&t=39655

### CODE REVIEW

**Comment by luzpaz** [comment]
Thanks for the PR! Though I wonder if the 'color' tab is the intuitive location for it? Perhaps this feature needs to be differentiated in to a separate tab ?

File: src/Gui/DlgSettings3DView.ui [hunk 1]
```diff
@@ -292,14 +292,58 @@ but slower response to any scene changes.</string>
...
+             <property name="minimumSize">
+               <size>
+                 <width>120</width>
+                 <height>0</height>
+               </size>
+             </property>
+             <property name="toolTip">
+               <string>
+                 Area for picking elements in 3D view.
+                 Larger value eases to pick things, but can make small features impossible to select.
```
Comment (luzpaz) [inline src/Gui/DlgSettings3DView.ui:313]: how does this tooltip look with all the extra space included on line 313 ?

Comment (arijeetbaruah) [reply src/Gui/DlgSettings3DView.ui:313]: good catch, I will resolve it as soon as possible

**Comment by arijeetbaruah** [comment]
@luzpaz
yes, the feature was in the Display->Color Tab. I moved it to Display->3D View as mention in the community forum [https://forum.freecadweb.org/viewtopic.php?f=8&t=39655](https://forum.freecadweb.org/viewtopic.php?f=8&t=39655)

**Comment by luzpaz** [comment]
Thanks @arijeetbaruah and wmayer!  
The wiki documentation needs to be updated, right ?  
https://wiki.freecadweb.org/Preferences_Editor#Display_settings

**Comment by arijeetbaruah** [comment]
well Ideally yes @luzpaz @wwmayer

**Comment by arijeetbaruah** [comment]
thanks for the merge @luzpaz @wwmayer

**Comment by luzpaz** [comment]
> well Ideally yes @luzpaz @wwmayer

lol, ok, I'll do it

**Pull request merged**